modules/currency-l0/src/main/scala/org/tessellation/currency/ExampleCurrencyL0.scala

```diff
@@ -0,0 +1,16 @@
+package org.tessellation.currency
+
+import java.util.UUID
+
+import org.tessellation.currency.schema.currency.TokenSymbol
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.cluster.ClusterId
+
+import eu.timepit.refined.auto._
+
+object ExampleCurrencyL0
+    extends CurrencyL0App(
+      TokenSymbol("TST"),
+      ClusterId(UUID.fromString("517c3a05-9219-471b-a54c-21b7d72f4ae5")),
+      Address("DAG77VVVRvdZiYxZ2hCtkHz68h85ApT5b2xzdTkn")
```

Comment:
Couldn’t this be just a set of CLI parameters?
```
java -jar currency-l1.jar —symbol=TST —clusterId=517… —identifier=DAG7Q…
```

Reply:
Yes, we plan to get rid of inheritance in favor of parametrized binary. For now I left it like that because @TheMMaciek also implemented it like that and the draft docs have it like that.

Also `clusterId` can be derived by calling something like `ClusterId(UUID.nameUUIDFromBytes(address.value.getBytes))`

Reply:
+ we can get rid of `TokenSymbol` because it isn't even used and is just for the debugging. According to latest agreements with Alex and Ryle - all the currencies will be isomorphic so there is no need to introduce symbol. At least for now.

Reply:
At least that's what I intend to do in the currency L1 part.

Reply:
For now we will use inheritance model because providing CLI parameters needs bigger refactoring.